layouts/shortcodes/region-param.html

```diff
@@ -0,0 +1,4 @@
+{{ if  not (.Get "key") }}{{ errorf "region-param shortcode error: Missing value for param 'key': %s" .Position }}{{ end }}
+{{ if eq (.Get "code") "true"  }}<code class="js-region-param" data-region-param="{{ .Get "key" }}"></code>
+{{ else }}<span class="js-region-param" data-region-param="{{ .Get "key" }}"></span>
+{{ end }}
```

Comment:
You are probably aware but thought I'd mention hyphens on on tags can sometimes help with [whitespace](https://gohugo.io/templates/introduction/#whitespace) too